domain/src/test/java/org/oppia/domain/audio/CellularAudioDialogControllerTest.kt

```diff
@@ -27,6 +27,7 @@ import org.mockito.junit.MockitoRule
 import org.oppia.app.model.CellularDataPreference
 import org.oppia.testing.TestCoroutineDispatchers
 import org.oppia.testing.TestDispatcherModule
+import org.oppia.testing.TestLogReportingModule
```

Comment:
Why are we importing these? Wont this get handled by the crashlytics controller PRs?

Reply:
Yes, it can be handled in the crashlytics implementation PR as well but that PR is based on this branch only. But as these test files have dependency over `TestLogReportingModule`, so had to import in this PR only.

Reply:
Got it, thanks!